google-cloud-core/src/main/java/com/google/cloud/ReadChannel.java

```diff
@@ -62,6 +63,23 @@ public interface ReadChannel extends ReadableByteChannel, Closeable, Restorable<
    * <p>If used in conjunction with {@link #seek(long)} the total number of returned bytes from this
    * channel will be reduced by the number of bytes specified to seek.
    *
+   * <p>The value provided as {@code limit} will define a <a target="_blank" rel="noopener
+   * noreferrer"
+   * href="https://en.wikipedia.org/wiki/Interval_(mathematics)#Classification_of_intervals">left-open,
+   * right-closed</a> interval along with either {@code 0} or any value provided to {@link
+   * #seek(long)}, i.e. {@code [}{@link #seek(long)}{@code , }{@link #limit(long)}{@code )}.
```

Comment:
Nit: you have [ and ) flipped; 

Left-open, right-closed: 
![image](https://user-images.githubusercontent.com/1907648/205401256-4da71f7b-8b1c-4c2a-9cb3-cf6dda020fb1.png)

Reply:
Thanks for catching this, I used the correct notation but the wrong name. It's supposed to be left-closed, right open so that `a <= x < b`. I'll push a fix.